R/SVG.R

```diff
@@ -1,14 +1,46 @@
 #' An SVG Graphics Driver
 #'
-#' This function produces graphics compliant to the current w3 svg XML standard.
-#' The driver output is currently NOT specifying a DOCTYPE DTD.
+#' This function produces graphics compliant to the current w3 svg XML
+#' standard. The driver output is currently NOT specifying a DOCTYPE DTD.
+#'
+#' svglite provides two ways of controlling fonts: system fonts
+#' aliases and user fonts aliases. Supplying a font alias has two
+#' effects. First it determines the \code{font-family} property of all
+#' text anchors in the SVG output. Secondly, the font is used to
+#' determine the dimensions of graphical elements and has thus an
+#' influence on the overall aspect of the plots.
```

Comment:
"This means that for optimal display, the font must be available on both the computer used to create the svg, and the computer used to render the svg. The defaults are fonts that are available on almost all systems: there may be small differences between them, but they are unlikely to cause problems in most causes"

And maybe describe what happens if the font isn't available on the server (i.e. fallback to font X), and what happens if the font isn't available to the user.